pubsub/api/PubsubTest.cs

```diff
@@ -335,6 +337,32 @@ public void TestListTopics()
         }
 
         [Fact]
+        public void TestListTopicPages()
+        {
+            string topicId = "testTopicForListingTopics";
+            string topicName = PublisherClient.FormatTopicName(_projectId, topicId);
+            for (int i = 0; i < 5; ++i)
+            {
+                CreateTopic($"{topicId}{i}", _publisher);
+            }
+            string projectName = PublisherClient.FormatProjectName(_projectId);
+            var pages = _publisher.ListTopics(projectName).AsPages();

```

Comment:
I don't think so in general. (We did for Datastore and Bigquery, I think.)

This would only change to AsResponses, in the current PR.